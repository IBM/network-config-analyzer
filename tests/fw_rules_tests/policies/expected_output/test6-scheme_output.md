|query|src_ns|src_pods|dst_ns|dst_pods|connection|
|---|---|---|---|---|---|
|connectivity_map, config: np6|
|||ip block: 0.0.0.0/0|[default,ibm-system-new,kube-system-new-dummy-to-ignore]|[*]|All connections|
||[default,ibm-system-new,kube-system-new,kube-system-new-dummy-to-ignore]|[*]||ip block: 0.0.0.0/0|All connections|
||[default,ibm-system-new,kube-system-new,kube-system-new-dummy-to-ignore]|[*]|[default,ibm-system-new,kube-system-new-dummy-to-ignore]|[*]|All connections|
||[kube-system-new]|[app in (helm,ibm-kube-fluentd,ibm-storage-watcher,vpn)]|[kube-system-new]|[*]|All connections|


