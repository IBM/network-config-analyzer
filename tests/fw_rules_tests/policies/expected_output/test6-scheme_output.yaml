- query: 'connectivity_map, config: np6'
  rules:
  - src_ip_block:
    - 0.0.0.0/0
    dst_ns:
    - default
    - ibm-system-new
    - kube-system-new-dummy-to-ignore
    dst_pods:
    - '*'
    connection:
    - All connections
  - src_ip_block:
    - ::/0
    dst_ns:
    - default
    - ibm-system-new
    - kube-system-new-dummy-to-ignore
    dst_pods:
    - '*'
    connection:
    - All connections
  - src_ns:
    - default
    - ibm-system-new
    - kube-system-new
    - kube-system-new-dummy-to-ignore
    src_pods:
    - '*'
    dst_ip_block:
    - 0.0.0.0/0
    connection:
    - All connections
  - src_ns:
    - default
    - ibm-system-new
    - kube-system-new
    - kube-system-new-dummy-to-ignore
    src_pods:
    - '*'
    dst_ip_block:
    - ::/0
    connection:
    - All connections
  - src_ns:
    - default
    - ibm-system-new
    - kube-system-new
    - kube-system-new-dummy-to-ignore
    src_pods:
    - '*'
    dst_ns:
    - default
    - ibm-system-new
    - kube-system-new-dummy-to-ignore
    dst_pods:
    - '*'
    connection:
    - All connections
  - src_ns:
    - kube-system-new
    src_pods:
    - (has(app) and app not in (ibm-file-plugin,ibm-keepalived-watcher))
    dst_ns:
    - kube-system-new
    dst_pods:
    - '*'
    connection:
    - All connections

